CHANGELOG.md

```
@@ -1,96 +1,94 @@
-#### 10.2.2 - October 2024
+#### 9.5.2 - October 2024
   * Added new `FieldEnum` with all the available field names
   * Code security improvements as per CodeQL recommendations
 
-#### 10.2.1 - June 2024
+#### 9.5.1 - June 2024
   * (PW-1913) Added IBAN validation for Egypt local account structure
   * Restore deprecated method in MT210 class
 
-#### 10.2.0 - May 2024
+#### 9.5.0 - May 2024
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)